apps/ewallet/lib/mix/tasks/omg.seed.ex

```diff
@@ -49,7 +49,11 @@ defmodule Mix.Tasks.Omg.Seed do
   #
 
   defp seed_spec([]) do
-    [{:ewallet_db, :seeds}]
+    [{:ewallet_db, :seeds}] ++ seed_spec(["--settings"])
```

Comment:
I think this should be part of the default seeds na (just add it to `seed_spec([])`'s list rather than a new function to handle it). `seed_spec([])` should be the "final" function to get called.

Reply:
What I tried to do here is give the option to only run the settings seeds for all our systems that are already deployed (thought it'd be nice :D). Totally fine to change it, but not sure I follow what you mean, can you give me some code?